multus_sriov_cni.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash -x
+
+export RECLONE=${RECLONE:-true}
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+export PATH=/usr/local/go/bin/:/opt/cni/bin/:/home_stack/cni/cni/cnitool/$PATH
+export CNI_PATH=/opt/cni/bin/
+
+MULTUS_CNI_BRANCH=${MULTUS_CNI_BRANCH:-master}
+MULTUS_CNI_REPO=${MULTUS_CNI_REPO:-https://github.com/intel/multus-cni}
+
+SRIOV_CNI_BRANCH=${SRIOV_CNI_BRANCH:-master}
+SRIOV_CNI_REPO=${SRIOV_CNI_REPO:-https://github.com/intel/sriov-cni}
+
+PLUGINS_BRANCH=${PLUGINS_BRANCH:-master}
+PLUGINS_REPO=${PLUGINS_REPO:-https://github.com/containernetworking/plugins.git}
+
+SRIOV_NETWORK_DEVICE_PLUGIN_REPO=${SRIOV_NETWORK_DEVICE_PLUGIN_REPO:-https://github.com/intel/sriov-network-device-plugin}
+SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH=${SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH:-remotes/origin/feature/device_selectors}
+
```

Comment:
https://github.com/Mellanox/sriov_cni_ci/blob/fa105c93c8af1f9447bc1f3e839398f347ce82c2/multus_sriov_cni.sh#L11-L21
move then all the exports